governance/polls/Prioritization Sentiment - Technical Resource Deployment - June 6, 2022.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Prioritization Sentiment - Technical Resource Deployment - June 6, 2022
+summary: Rank your Technical Resource Priority Sentiment to indicate where you believe MakerDAO should be focusing resources. 
+discussion_link: https://forum.makerdao.com/t/prioritization-framework-sentiment-polling/15554
+vote_type: Ranked Choice IRV
+categories:
+   - Misc Governance
+   - Priority Sentiment
```

Comment:
You will need to include this on the tags sheet as well - categories here are no longer read.